test/fluree/db/query/history_test.clj

```diff
@@ -959,3 +959,50 @@
                           (-> (get c "f:commit")
                               (update "f:data" select-keys ["f:t"])
                               (select-keys ["f:author" "f:txn" "f:data"]))))))))))
+
+(deftest ^:integration txn-annotation
```

Comment:
Could you add some English documentation to this test using both the `testing` macro to explain what's being tested, and the second argument to the `is` macro to describe the expected outcome?